connector/google/google.go

```diff
@@ -76,8 +87,7 @@ type googleConnector struct {
 	cancel        context.CancelFunc
 	logger        logrus.FieldLogger
 	hostedDomains []string
```

Comment:
We should make this just `hostedDomain string` since we are only working with a single hostedDomain and makes things a little cleaner in the `getGroups` function.